postgresDb/dummyData.js

```diff
@@ -0,0 +1,95 @@
+const movies = require('./index.js');
+
+
+// Helper functions for generating movie objects:
+
+function generateProfileValues() {
+  // number of genres to calculate values for (a value between 1 and 14)
+  const genreNum = Math.floor(Math.random() * 13) + 1;
+  // generate number of random values equal that add up to 100
+  const values = [];
+  let total = 0;
+  for (let i = 0; i < genreNum; i += 1) {
+    const newValue = Math.floor(Math.random() * 100);
+    values.push(newValue);
+    total += newValue;
+  }
+  let newTotal = 0;
+  for (let j = 0; j < values.length; j += 1) {
+    const newValue = Math.floor(100 * (values[j] / total));
+    values[j] = newValue;
+    newTotal += newValue;
+  }
+  // add final value so that they add up to 100
+  values.push(100 - newTotal);
+  return values;
+}
+
+function assignProfileValues() {
+  // default all values to 0
+  const profile = [
```

Comment:
If you have time (not super vital, since I assume this is working right now), maybe refactor to use an object, just for the sake of cleanliness.

Reply:
done